features/docs/cli/fail_fast.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Fail fast
+
+  The --fail-fast flag causes Cucumber to exit immediately after the first 
+  scenario fails.
+
+  Scenario: When a scenario fails
```

Comment:
The annotation `@spawn` needs to be added to this scenario so that the cucumber instance started by the `When` step is not executed in the same process as the cucumber instance that rake has started. This problem reminds me of "singletons considered harmful" and a quote from one of the pages return from a search of that topic: "everything that can be done with Singletons, can be done better with dependency injection". @mattwynne `Cucumber.wants_to_quit` really?